### PR TITLE
Capstone Project Ready to Be Released

### PR DESCRIPTION
**Pull request by rpire**
# Project Requirements:
## API
- Fetched data from the API and displayed it in the web app using the `Redux` store.
## Design
- Followed *Nelson Sakwa's* design to style the whole web app.
- Acknowledged *Nelson Sakwa* in the `README.md` file of this project.
## Home page
- Displayed a list of regions and numeric data obtained from the API.
- Implemented an option to filter regions by their name.
- Implemented navigation to each region on click.
## Details page
- Retrieved detailed data from the API for each region.
- Implemented a return button in the `nav` of each region page.
## Testing
- Built a test file for each component.
- Built a test file for functions.
## Deployment
- Deployed the project using GitHub pages and included the link in the `README.md` file.
- Live demo link [here](https://rpire.github.io/covid-metrics/).
## Presentation
- Recorded a video using Loom and included the link in the `README.md` file.
- Presentation link [here](https://www.loom.com/share/7b3c1779d72a4ec5ac60c28778f2064e).

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
```
Comment (henatan99) [inline src/components/App.js:1]: - [ ] The `App` component is a top-level component and document structure-wise it is more logical to put this file in the `src` folder. Kindly make changes here. :pray:

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    .filter((region) => region.name.toLowerCase()
+      .includes(search.toLowerCase()));
+
+  return (
+    <>
+      <HomeNavbar
+        search={search}
+        handleChange={handleChange}
+      />
+      <div className="main-container flex-box">
```
Comment (henatan99) [inline src/pages/Home.js:24]: - [ ] For better sematic, I think a `section` tag would serve better here. :pray:

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          <li key={region.id}>
+            <Link to={`/${region.id}`}>
+              <Region
+                region={region}
+                id={filteredRegions.indexOf(region)}
+              />
+            </Link>
+          </li>
+        ))}
+      </ul>
```
Comment (henatan99) [inline src/pages/Home.js:48]: - [ ] I also recommend you wrap these two elements in a `section` tag. Don't you think it gives more meaning semantically? More semantic HTML always helps in AI-based search and visually impaired people. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @rpire , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

- Honestly, you did above and beyond the mandatory requirements. Very impressive work. :champagne: :+1: :1st_place_medal: . Just make some little changes as there is always room for improvement.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home.js (hunk 2 above)
Comment (rpire) [reply src/pages/Home.js]: Required change completed.

File: src/pages/Home.js (hunk 3 above)
Comment (rpire) [reply src/pages/Home.js]: Required change completed.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @rpire,

Good job on implementing the suggested changes. Well Done.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
Hello, @Dipeshtwis 👋

Thank you for reviewing my work!

I will proceed to merge these changes into the `main` branch.  🚀

Cheers and Happy coding 🤓

**Pull request merged**